core/src/main/java/com/datastrato/gravitino/catalog/FilesetOperationDispatcher.java

```diff
@@ -43,9 +45,20 @@ public FilesetOperationDispatcher(
    */
   @Override
   public NameIdentifier[] listFilesets(Namespace namespace) throws NoSuchSchemaException {
+    NameIdentifier standardizedSchemaIdent =
+        standardizeNameIdentifier(NameIdentifier.of(namespace.levels()), Capability.Scope.SCHEMA);
+    Namespace standardizedNamespace =
+        Namespace.of(
+            standardizedSchemaIdent.namespace().level(0),
+            standardizedSchemaIdent.namespace().level(1),
+            standardizedSchemaIdent.name());
```

Comment:
Can we have a wrapper method like `doWithCatalog` to standardize the input namespace or nameidentifier, like:

```java
doWithStandardizedIdent { ident ->
   doWithXXX
 }
```

Reply:
The current change placed everywhere makes code hard to review.

Reply:
I think you didn't change this, do you have a good solution?